.env

```diff
@@ -12,12 +12,13 @@ POSTGRES_USER=postgres
 POSTGRES_PASSWORD=password
 
 # Docker Image tags
-MANAGEMENT_VERSION=v1.4.1
-BLACKLIGHT_VERSION=v1.3.0
-POSTGRES_VERSION=v1.0.0
+IIIF_MANIFEST_VERSION=v2.0.0
```

Comment:
It might be better in the future to make changes like the container version updates in a separate PR since they're a separate concern not required to add the Camerata version to the dashboard.